src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/JDBCEncryptionDecryptionTest.java

```diff
@@ -134,10 +126,10 @@ public void testBadAkv(String serverName, String url, String protocol) throws Ex
 
         try {
             SQLServerColumnEncryptionAzureKeyVaultProvider akv = new SQLServerColumnEncryptionAzureKeyVaultProvider(
-                    (SQLServerKeyVaultAuthenticationCallback) null);
+                    (TokenCredential) null);
             fail(TestResource.getResource("R_expectedExceptionNotThrown"));
-        } catch (SQLServerException e) {
-            assertTrue(e.getMessage().matches(TestUtils.formatErrorMsg("R_NullValue")));
+        } catch (NullPointerException exception) {
```

Comment:
Driver should not throw a NullPointerException . Better add a null check to the constructor and throw SQLServerException .